README.md

```diff
@@ -1,3 +1,10 @@
+[![Unit tests workflow](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/workflows/Unit%20tests%20workflow/badge.svg)](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/actions)
+[![E2E tests workflow](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/workflows/E2E%20tests%20workflow/badge.svg)](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/actions)
```

Comment:
wondering, should we change this to `Integration tests workflow`? Also, since it will display as passing/failing, should we just have it say `Unit tests` and `Integration tests`?

Reply:
Sounds good to me. Will change it.